.github/workflows/generate_test_files.yml

```
@@ -15,7 +15,7 @@ jobs:
       matrix:
         include:
           - { name: pynwb-1.5.1, pynwb-version: "1.5.1", python-version: "3.8"}
-          - { name: pynwb-2.1.0, pynwb-version: "2.1.0", python-version: "3.9"}
+          - { name: pynwb-2.1.0, pynwb-version: "2.1.0", python-version: "3.10"}
```

```suggestion
          - { name: pynwb-2.1.0, pynwb-version: "2.1.0", python-version: "3.9"}
```

This workflow is to demo generating older test files for backwards compatibility. Technically pynwb 2.1.0 did not support python 3.10 so think this should remain 3.9.